### PR TITLE
DTSCI-102 Add generic Metadata component to pages

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Added component MetaData.js to take pageProps and render metadata in English or French. 
- Replaced old <Head> calls with calls to new component <MetaData>
- Created new prop meta in each page to hold static metadata information as a placeholder. 

### What to test for/How to test
See calls to getStaticProps on pages to view metadata information.

### CODE REVIEW

**Review by P0NDER0SA — APPROVED** [review]
Nice work, looks pretty good!

File: pages/404.js [hunk 1]
```diff
@@ -55,7 +55,23 @@ export default function Custom404() {
 }
 
 export async function getStaticProps({ locale }) {
+  /* Place-holder Meta Data Props */
+  const meta = {
```
Comment (nibivi77) [inline pages/404.js:59]: Just a question, why do we have this if we are not using it? Maybe we should add it to the page?

File: pages/_error.js [hunk 2]
```diff
@@ -62,11 +62,29 @@ function CustomError({ statusCode }) {
   )
 }
 
-Error.getInitialProps = ({ res, err, locale }) => {
+export async function getServerSideProps({ res, err, locale }) {
   const statusCode = res ? res.statusCode : err ? err.statusCode : 404
+
+  /* Place-holder Meta Data Props */
+  const meta = {
```
Comment (nibivi77) [inline pages/_error.js:69]: Same thing here, maybe we should include it on the page?

File: pages/home.js [hunk 3]
```diff
@@ -17,8 +17,25 @@ export default function Home(props) {
 export async function getStaticProps({ locale }) {
   const content = await fetchContent()
   const langToggleLink = locale === 'en' ? '/fr/home' : '/home'
+
+  /* Place-holder Meta Data Props */
+  const meta = {
```
Comment (nibivi77) [inline pages/home.js:22]: Do we need this one here since we are passing it to Layout.js?

**Review by nibivi77 — COMMENTED** [review]
This is good than Curtis. Left a few comments :)

**Review by nibivi77 — APPROVED** [review]
Nevermind about my useless comments! Good work :)

**Pull request merged**